optaplanner-core/src/main/java/org/optaplanner/core/impl/score/stream/drools/common/rules/AbstractRuleAssembler.java

```diff
@@ -105,7 +105,11 @@ protected static void impactScore(DroolsConstraint constraint, Drools drools, Ab
             BigDecimal impact) {
         RuleContext kcontext = (RuleContext) drools;
         constraint.assertCorrectImpact(impact);
-        scoreHolder.impactScore(kcontext, impact);
+        try {
+            scoreHolder.impactScore(kcontext, impact);
+        } catch (UnsupportedOperationException exception) {
+
+        }
```

Comment:
If we want to do this (and the jury is out on that), then we 'd want to comment here clearly why we're sweeping this under the mat.

Reply:
Yeah, sorry, this catch shouldn't be there at all. :-)